tests/migration_test.go

```diff
@@ -3642,18 +3649,10 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			// Need to wait for cloud init to finnish and start the agent inside the vmi.
 			tests.WaitAgentConnected(virtClient, vmi)
 
-			runStressTest(vmi, stressDefaultVMSize, stressDefaultTimeout)
-
 			// execute a migration, wait for finalized state
 			By("Starting the Migration")
 			migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
-			migrationUID := runMigrationAndCollectMigrationMetrics(vmi, migration, 180)
-
-			// check VMI, confirm migration state
-			tests.ConfirmVMIPostMigration(virtClient, vmi, migrationUID)
```

Comment:
Why do we delete this? I am not sure I understand the concern here.

Reply:
As I wrote in the description of https://github.com/kubevirt/kubevirt/pull/8586/commits/a75a440616fc58ac0a5b08eebe607c2abc636422:

> It's hard to find the exact timeout / bandwidth limitation configs to
ensure the migration both ends on one hand, and is occuring for enough
time to collect metrics about it. Since the test is not focused on
validating the migration should end, this shouldn't be part of the test
(also, there are many other tests are focused on validating a migration
finish).
>
> In addition, Eventually clause is now changed to keep trying to get
valid metrics of some of them are with a zero-value, something that is
acceptable in Prometheus' context.

TLDR: this isn't what the test focuses on and aims to validate in the first place, and it's complicating the test and causing it to be much less stable and more flaky. There are many other tests that are focused on that goal.

Reply:
I understand that part but did we see this "waiting for migration end" part to be flaky?

Reply:
Yes. From local testing I see that the migration sometime ends and sometimes the timeout exceeds. Essentially this is a matter of tuning the timeout and bandwidth parameters which is outside the scope of this test IMHO. We have a lot of other tests that are focused on migration getting converged.